src/mezz/base-series.r

```diff
@@ -233,7 +255,27 @@ trim: function [
         ]
 
         any-string? series [
-            rule: make bitset! if with [str] else [reduce [space tab]]
+            ; These are errors raised by the C version of TRIM in R3-Alpha.
+            ; One could question why /with implies /all.
+            ;
+            if any [
+                all [
+                    auto
+                    any [head_TRIM tail_TRIM lines]
+                ]
+                all [
+                    any [all_TRIM with]
+                    any [auto head_TRIM tail_TRIM lines]
+                ]
+            ][
+                fail "Invalid refinements for TRIM of STRING!"
+            ]
```

Comment:
Continued food for thought on the place of things like AND, OR, THEN...as I want people trying them and getting experience with them:

    any [
        auto and (any [head_TRIM tail_TRIM lines])
        any [all_TRIM with] and (any [auto head_TRIM tail_TRIM lines])
    ] then [
        fail "Invalid refinements for TRIM of STRING!"
    ]

Does the mixture make the intent more clear?  Wondering what kinds of thoughts it brings about.